webui/react/src/pages/DesignKit.tsx

```diff
@@ -56,6 +56,116 @@ import useConfirm, { voidPromiseFn } from '../components/kit/useConfirm';
 import css from './DesignKit.module.scss';
 import { CheckpointsDict } from './TrialDetails/F_TrialDetailsOverview';
 
+const themeStatus = [
```

Comment:
These shouldn't be defined in the demo page, they should be accessible to the rest of the design kit as enumerations.